examples/next/faust-nx/pages/preview.tsx

```diff
@@ -0,0 +1,12 @@
+import 'faustnx.config';
+import { getWordPressProps, WordPressTemplate } from 'faust-nx';
+import { GetServerSidePropsContext } from 'next';
+import client from 'client';
+
+export function getServerSideProps(ctx: GetServerSidePropsContext) {
+  return getWordPressProps({ client, ctx });
```

Comment:
do we need to use `getWordPressProps` as we perform everything on the client side?

Reply:
Good catch, this was a remnant from a prior server side attempt.